app/code/Magento/Customer/Controller/Address/FormPost.php

```diff
@@ -89,18 +96,44 @@ public function __construct(
             $dataProcessor,
             $dataObjectHelper,
             $resultForwardFactory,
-            $resultPageFactory
+            $resultPageFactory,
+            $customerRepository
         );
     }
 
     /**
      * Extract address from request
      *
-     * @return \Magento\Customer\Api\Data\AddressInterface
+     * @return AddressInterface
+     * @throws \Exception
+     * @deprecated Use extractCurrentAddress instead
+     */
+    protected function _extractAddress(): AddressInterface
```

Comment:
Please, make this method private and avoid using underscores in method names

Reply:
Btw this is an original `protected function _extractAddress()` method